### ISSUE TITLE
Status code on returns

### ISSUE THREAD

**Issue by sixcolors** [Yash294/TODO#1]
https://github.com/Yash294/TODO/blob/518aefdc7c4bafaa503c8419795a537eeea3378c/controllers/user.go#L38

That will return JSON with a status code of 200 or Ok, or in other words Success.

try something like this

```
		 return c.Status(fiber.StatusInternalServerError).JSON(fiber.Map{
			"success": false,
			"message": "Failed to check acting user permissions",
		})
```

**Comment by sixcolors** [comment]
https://docs.gofiber.io/api/ctx

**Comment by Yash294** [comment]
Resolved in ddfc7920e0655aee3408dd06101f0078c0fb920a
